internal/v1/api.yaml

```diff
@@ -1069,6 +1070,10 @@ components:
           $ref: '#/components/schemas/Customizations'
         metadata:
           $ref: '#/components/schemas/BlueprintMetadata'
+        custom_repositories:
+          type: array
+          items:
+            $ref: '#/components/schemas/CustomRepositoryExported'
```

Comment:
I have decided to add a new column here. We are reusing the Customizations field in the API for BlueprintExportResponse, and I did not like creating a whole new "CustomizationsExport" struct that would be the same, just with different type for Custom Repositories.
Other option we have, is to make the Customizations.CustomRepositories into
`
      oneOf:
            $ref: '#/components/schemas/CustomRepository'
            $ref: '#/components/schemas/CustomRepositoryExported'
`
But honestly, I do not like that solution either. I think that we can handle one additional field here. Feel free to express your thoughts on this.

Reply:
I don't really mind the additional field here, but few thoughts
1) we now have one custom_repositories under customizations and one outside, I think it is quite confusing
2) it changes the blueprint structure, so now it won't work in the import, will it?

Reply:
1. Yes, I agree, we could do something like `custom_repositories_exported`. And we can even set the pointer to custom repositories to nil, in order to not have two custom_repositories. But I do not think it really matters.
2. It should not break anything, we ignore any extra fields. The idea was to make the import as easy as possible without creating complications. If we ever get to the point where export will be stabilized, I think we can start to be a bit more strict for import files.

Reply:
Have you considered adding it to `BlueprintMetadata` component which already holds parent id and export time and date?

Another option is to simply extend the blueprint used in image builder, it already has a reference to OpenSCAP hosted resource anyway so there is the pattern already.

@croissanne asked for explanation below, so I suggest to discuss this with her.

Reply:
I did consider it. We could also put it as a separate field into customizations, something like "CustomRepositoriesExported". However, the point of the new field outside existing ones is that we will not need this field anywhere else other than in the export structure. This will just appear in the file that the users will be able to download anywhere else.
So I do not want the normal "blueprint" structure to have a new field because it would not be used anywhere. It will be a temporary field that lives only in the json file.
1) We export custom repos, including the response from /repositories/bulk_export/ into the export json
2) On import, we call the /repositories/bulk_import/ with this field in order for custom services to create these repositories for us. We can build the "custom_repositories: [xxx, xxx]" field because the account will finally have the given repositories saved. Once we do that, we throw away the CustomRepositoriesExported.

Reply:
Makes sense. LGTM